<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newhard</class>
 <widget class="QDialog" name="newhard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="soket">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>161</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1151</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1155</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FM2+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FM3+</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="soketl">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>142</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cокет</string>
   </property>
  </widget>
  <widget class="QComboBox" name="graph">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>161</y>
     <width>136</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Intel HD Graphics 3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Intel HD Graphics 4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radeon R7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Немає</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="grapl">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>142</y>
     <width>132</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вбудований графічний чіп</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cash">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>206</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="cashl">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>187</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кеш(МБ)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freq">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>206</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="freql">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>187</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Частота</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="Imagel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Зображення</string>
   </property>
  </widget>
  <widget class="QLabel" name="cernell">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>232</y>
     <width>74</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість ядер</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cernel">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>251</y>
     <width>32</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="pcxv">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.0</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="pcxx">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="pcxxl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="pcxvl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Версія PCIe</string>
   </property>
  </widget>
  <widget class="QComboBox" name="canal_type">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DDR3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DDR4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="canal_typel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип памяті</string>
   </property>
  </widget>
  <widget class="QComboBox" name="canal_count">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="canal_countl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість входів памяті</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DirectX">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="DirectXl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>DirectX</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OpenGL">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="OpenGLl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenGL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CFreq">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="MFreq">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CFreql">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Частота ядра</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mfreql">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Частота памяті</string>
   </property>
  </widget>
  <widget class="QComboBox" name="memVc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="MVcl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Обсяг памяті(Гб)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="memory">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="memoryl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Обсяг памяті</string>
   </property>
  </widget>
  <widget class="QComboBox" name="memtype">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HDD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SSD</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="memtypel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип памяті</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxmem">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимум ОЗУ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="maxmemm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>125</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Процесор</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Материнська плата</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>ОЗУ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Жорсткий диск</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>Відеокарта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>ПЗ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>278</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="namel">
        <property name="text">
         <string>Назва</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
      <item>
       <widget class="QLabel" name="yearl">
        <property name="text">
         <string>Рік</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="year">
        <item>
         <property name="text">
          <string>2007</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2008</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2009</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2010</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2011</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2012</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2013</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2014</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2015</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2016</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2017</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="creatorl">
        <property name="text">
         <string>Виробник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="creator"/>
      </item>
      <item>
       <widget class="QLabel" name="pricel">
        <property name="text">
         <string>Ціна</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="price">
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="bestcpu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bestvc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="BestOzu">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="bestl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Рекомендовані вимоги</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mincpu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="minvc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="minozu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Minl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Мінімальні вимоги</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newhard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newhard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
